src/DataBase/GoalDatabaseSql.cs

```diff
@@ -170,26 +170,25 @@ public async Task<int> MoveGoalToExpensesAsync(int goalId, int userId, IEntry ex
             {
                 CommandType = CommandType.Text
             };
+
             sqlCommand.Parameters.AddWithValue("@Id", goalId);
             sqlCommand.Parameters.AddWithValue("@UserId", userId);
             await sqlCommand.ExecuteNonQueryAsync();
 
             var dbExpense = new ExpenseModel(expense, userId);
-            sqlCommand =
-                new SqlCommand(
-                    "INSERT INTO Expenses(UserId, Amount, Title, Date, IsMonthly, Importance, Currency) VALUES (@UserId, @Amount, @Title, @Date, @IsMonthly, @Importance, @Currency);SELECT CAST(scope_identity() AS int);",
-                    sqlConnection)
-                { CommandType = CommandType.Text };
+            sqlCommand = new SqlCommand(
+                "INSERT INTO Expenses(UserId, Amount, Title, Date, IsMonthly, Importance, Currency) " +
+                "VALUES (@UserId, @Amount, @Title, @Date, @IsMonthly, @Importance, @Currency);SELECT CAST(scope_identity() AS int);",
```

Comment:
You can format string using verbatim string literal, like in answer to this [question](https://stackoverflow.com/questions/1100260/multiline-string-literal-in-c-sharp).